### PR TITLE
Changed readme link from google form

### PR DESCRIPTION
**Pull request by cliftonfowler**
Changed readme link from google form to new link to create a developer account document

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ BuzzwordCRM is a sample application that demonstrates how to build applications
 
 ## Running the app locally
 
-1. Register as a Calendly developer (if you haven't already) by logging into your Calendly account and [completing this form](https://docs.google.com/forms/d/e/1FAIpQLSc9ltoPU9I_yyQt1gWLm6fa0xMhpPWm-mL_vfPfeilC_s1vTA/viewform) (note: this app assumes that your redirect uri will be http://localhost:3000/oauth/callback). We’ll process your request within 1 day and email you when it’s time to complete the setup.
+1. Register as a Calendly developer (if you haven't already) by logging into your Calendly account and [Create a developer account](https://developer.calendly.com/create-a-developer-account) (note: this app assumes that your redirect uri will be http://localhost:3000/oauth/callback). We’ll process your request within 1 day and email you when it’s time to complete the setup.
```
Comment (anolson) [inline README.md:7]: Should we remove the parts about "logging into your Calendly account" and  "We’ll process your request within 1 day and email you when it’s time to complete the setup."

```suggestion
1. Register as a Calendly developer (if you haven't already) [Create a developer account](https://developer.calendly.com/create-a-developer-account) (note: this app assumes that your redirect uri will be http://localhost:3000/oauth/callback).
```

Comment (RSalaam) [reply README.md:7]: Yes, I think we should remove mention of logging in and the approval process, as Andrew mentions above!

**Review by RSalaam — CHANGES_REQUESTED** [review]
I think we should remove mention of logging into dev account, and the approval process, as Andrew points out.

File: README.md (hunk 1 above)
Comment (cliftonfowler) [reply README.md]: Good find! Removing it

**Review by RSalaam — APPROVED** [review]
Thank you for updating!

**Review by anolson — APPROVED** [review]

**Pull request merged**